src/components/Dropdown/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+export const Dropdown = (props) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
rather than this ignore, I would suggest you to add [propTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) to this component, so that whoever uses it will have some guidelines